scripts/west_commands/blobs.py

```diff
@@ -55,6 +55,8 @@ def do_add_parser(self, parser_adder):
             - type: type of blob
             - version: version string
             - license_path: path to the license file for the blob
+            - license-abspath: absolute path to the license file for the blob
+            - license-click-through: need license click-through or not
```

Comment:
Is `click-through` clear on what the purpose is?

Reply:
Yes. I'm okay to use click-through.
Thanks.

Reply:
That's not what I meant, I find the wording of `click-through` a bit odd, maybe `license-agreement-required` or something is a better fit?

Reply:
@pdgendt clickthrough or clickwrap are known licensing terms where the underlying license requires presenting and obtaining consent to the terms before downloading. It is better to stick with known terminology. 

https://en.wikipedia.org/wiki/Clickwrap